mobile.css

```diff
@@ -0,0 +1,110 @@
+.body {
+    max-width: 375px;
+    margin: 0;
+}
+.nav__link {
+    display: none;
+}
+.logo {
+    width: 138px;
+    height: 42px;
+    margin: 5px 2px;
+}
+.vectors {
+    display: none;
+}
+.nav {
+    display: flex;
+    align-items: center;
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/Tesla-n/pullRequest?prid=5749910)